packages/office-ui-fabric-react/src/components/Persona/PersonaConsts.ts

```diff
@@ -8,6 +8,7 @@ import styles from './Persona.scss';
 export const PERSONA_SIZE = {
   [PersonaSize.tiny]: 'ms-Persona--tiny ' + styles.rootIsTiny,
   [PersonaSize.extraExtraSmall]: 'ms-Persona--xxs ' + styles.rootIsExtraExtraSmall,
+  [PersonaSize.size28]: 'ms-Persona--xxs ' + styles.rootIs28,
```

Comment:
Should be `ms-Persona--28`, although I'm unsure of mixing numbers with names like "small, tiny, extra small, etc.".